rlclientlib/live_model.cc

```diff
@@ -127,36 +136,73 @@ namespace reinforcement_learning
   int live_model::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response& resp, api_status* status)
   {
     INIT_CHECK();
-    return _pimpl->request_multi_slot_decision(context_json, flags, resp, status);
+    return _pimpl->request_multi_slot_decision(context_json, flags, resp, live_model::default_baseline_vector, status);
   }
 
   int live_model::request_multi_slot_decision(const char * context_json, multi_slot_response& resp, api_status* status)
   {
     return request_multi_slot_decision(context_json, action_flags::DEFAULT, resp, status);
   }
 
+  int live_model::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response& resp, const int* baseline_actions, size_t baseline_actions_size, api_status* status)
+  {
+    INIT_CHECK();
+    std::vector<int> baseline_vector = _c_array_to_vector(baseline_actions, baseline_actions_size);
+    if (event_id == nullptr && flags == NULL)
+    {
+      return _pimpl->request_multi_slot_decision(context_json, action_flags::DEFAULT, resp, baseline_vector, status);
+    }
+    if (event_id == nullptr)
+    {
+      return _pimpl->request_multi_slot_decision(context_json, flags, resp, baseline_vector, status);
+    }
+    if (flags == NULL)
```

Comment:
We call into this via the `LiveModelRequestMultiSlotDecisionWithBaselineAndFlags` as far as I can see, could `flags` be passed in here explicitly as `NULL` ?

Reply:
Good point. As discussed, from the c# layer flags is an enum which is not nullable. For anyone calling directly into the c++ layer NULL == 0 == ActionFlags::DEFAULT so this is indeed an unnecessary check. Will remove